R/reduce.R

```diff
@@ -75,14 +75,17 @@ reduce_right <- function(.x, .f, ..., .init) {
 #'     ggtitle("Simulations of a random walk with drift")
 #' }
 accumulate <- function(.x, .f, ..., .init) {
-  f <- as_function(.f, ...)
+  .f <- as_function(.f)
```

Comment:
I think you should add a comment here, briefly describing the reason why `as_function` doesn't take `...` here (unlike every other function).  

Equally, you could leave it, since adding it does no harm.